experiment/resources/dispatcher-startup-script-template.sh

```diff
@@ -26,6 +26,7 @@ docker run --rm \
   -e POSTGRES_PASSWORD={{postgres_password}} \
   -e CLOUD_SQL_INSTANCE_CONNECTION_NAME={{cloud_sql_instance_connection_name}} \
   -e DOCKER_REGISTRY={{docker_registry}} \
+  --shm-size=1g \
```

Comment:
Fuzzing isn't done in this image btw. What's the default here? Because this instance usually has hundreds of GB of RAM

Reply:
Thanks! Yep, I can confirm that removing this won't matter.
I am not sure how to check the default instance, though. Where is the value of `docker_registry` defined?

Reply:
I think the default value is 64mb on my local machine. If this doesn't any fuzzing and hasn't caused any problems, we can just skip this one.